src/treetensornetworks/projttns/projttnsum.jl

```diff
@@ -33,35 +33,18 @@ incident_edges(P::ProjTTNSum) = incident_edges(terms(P)[1])
 
 internal_edges(P::ProjTTNSum) = internal_edges(terms(P)[1])
 
-function product(P::ProjTTNSum, v::ITensor)::ITensor
-  Pv = product(terms(P)[1], v)
-  for n in 2:length(terms(P))
-    Pv += product(terms(P)[n], v)
-  end
-  return Pv
-end
+product(P::ProjTTNSum, v::ITensor) = noprime(contract(P, v))
```

Comment:
Can't this just be removed entirely in favor of the `AbstractProjTTN` version?

Reply:
`ProjTTNSum` is not a subtype of `AbstractProjTTN` if I am not mistaken.

Reply:
```
julia> supertypes(ITensorNetworks.ProjTTNSum)
(ProjTTNSum, Any)

julia> supertypes(ITensorNetworks.ProjTTN)
(ProjTTN, ITensorNetworks.AbstractProjTTN, Any)
```

Reply:
Oh that is strange, seems like it should be.